04.ls/ls_command.rb

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+COL_COUNT = 3
+
+def main
+  all_files = find_all_files
+  visible_files = delete_hiddenfile(all_files)
```

Comment:
```suggestion
  visible_files = select_visible_files(all_files)
```

名前は左辺と右辺をあわせた方がベターです
https://bootcamp.fjord.jp/pages/readable-variable-name

Reply:
動画で解説してくださり、ありがとうございます🙏
どんな点に留意して命名するのか、リファクタリングのポイントのイメージがついてわかりやすかったです。
コードを修正しましたので、ご確認よろしくお願いします。